.github/labeler.yml

```diff
@@ -48,6 +48,14 @@ python:
   - changed-files:
       - any-glob-to-any-file:
           - "**/*.py"
+shell script:
+  - changed-files:
+      - any-glob-to-any-file:
+          # If this project has any shell scripts that do not end in the ".sh"
+          # extension, add them below.
+          - "**/*.sh"
+          - bump-version
+          - setup-env
```

Comment:
We should either add the `.sh` extension to the file or explicitly list `terraform/scripts/toggle_prevent_destroy`. I don't _think_ there are any additional shell scripts that don't have the `.sh` extension.

Reply:
Please see commit 8900935.